exporter/serverstatus_collector_test.go

```diff
@@ -63,3 +64,26 @@ mongodb_metrics_commands_connPoolSync_failed 0` + "\n")
 	err := testutil.CollectAndCompare(c, expected, filter...)
 	assert.NoError(t, err)
 }
+
+func TestAllServerStatusDataCollector(t *testing.T) {
+	if inGithubActions() {
```

Comment:
Couldn't we make this test stable in github actions?

Reply:
No